src/components/NavBar/locale.ts

```diff
@@ -1,9 +1,15 @@
 const locale: Locale = {
   ko: {
     about: '이용안내',
+    petition: '청원하기',
+    all: '모든 청원',
+    answered: '답변된 청원',
   },
   en: {
-    about: 'about',
+    about: 'About',
+    petition: 'Make Petition',
```

Comment:
```suggestion
    petition: 'Petition',
```

Petition 자체가 동사기도 해서 그대로 쓰는건 어떨까 하는 생각,,

Reply:
요건 빠르게 투표를 받아봅시다!